docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -369,7 +369,7 @@ To find all EdgeX snaps on the public Snap Store, [search by keyword](https://sn
 
 ### Platform Snap
 [platform snap]: #platform-snap
-| [Installation][edgexfoundry] | [Configuration] | [Managing Services] | [Debugging] | [Source](https://github.com/edgexfoundry/edgex-go/tree/{{edgexversion}}/snap) |
```

Comment:
Snap code is removed.being removed so these links need to be removed until the Snap docs are removed.